Signal/src/network/PushManager.m

```diff
@@ -379,12 +402,38 @@ - (UIUserNotificationCategory *)signalMissedCallCategory
     callBackAction.destructive = NO;
     callBackAction.authenticationRequired = YES;
 
-    UIMutableUserNotificationCategory *callCategory = [UIMutableUserNotificationCategory new];
-    callCategory.identifier = PushManagerCategoriesMissedCall;
-    [callCategory setActions:@[ callBackAction ] forContext:UIUserNotificationActionContextMinimal];
-    [callCategory setActions:@[ callBackAction ] forContext:UIUserNotificationActionContextDefault];
+    UIMutableUserNotificationCategory *missedCallCategory = [UIMutableUserNotificationCategory new];
+    missedCallCategory.identifier = PushManagerCategoriesMissedCall;
+    [missedCallCategory setActions:@[ callBackAction ] forContext:UIUserNotificationActionContextMinimal];
+    [missedCallCategory setActions:@[ callBackAction ] forContext:UIUserNotificationActionContextDefault];
 
-    return callCategory;
+    return missedCallCategory;
+}
+
+- (UIUserNotificationCategory *)signalRejectedCallWithUnseenIdentityChangeCategory
```

Comment:
Well - sort of. 

UserNotificationsAdaptee is currently unused code, but it's iOS10 only. I've been trying to keep it up do date as we add new actions so that the eventual transition to UserNotifications is easier. But even then, we'll still need to have this pre iOS10 implementation.